docs/user/03-21-create-service-instance-with-custom-secret.md

```diff
@@ -0,0 +1,85 @@
+# Create a Service Instance with a Custom Secret
+
+To have multiple service instances from different subaccounts associated with one namespace, you must use a custom Secret to create these service instances.
+
+## Context
+
+To create a service instance with a custom Secret, you must use the **btpAccessCredentialsSecret** field in the `spec` of the service instance. In it, you pass the Secret from the `kyma-system` namespace to create your service instance. You can use different Secrets for different service instances.
+
+## Prerequisites
+
+* A subaccount in the SAP BTP cockpit
+
+## Procedure
+
+### Create Your Custom Secret
```

Comment:
In general, I'm missing where and how users should create service instances, bindings. Should it be done from the cockpit, dashboard? Somewhere else?

Reply:
In lines 83-84, there are links to instructions on creating SIs and SBs. Do you think I should move them up to the procedure?

Reply:
Or you can add inline links. For example: Create an [SAP Service Manager service instance](https://github.com/kyma-project/btp-manager/blob/f4021bb32ab14704ae4dee398cd9eb1bb536cf42/docs/user/03-30-management-of-service-instances-and-bindings.md#create-a-service-instance) with the `service-operator-access` plan